### DISCUSSION TITLE
Attention_type='0010'?

### DISCUSSION THREAD

**Discussion by lijoe123** [open-mmlab/mmdetection#8977] [Q&A]
Could you please tell me the Attention_type mean? May i make the attention_type='1101'? 

https://github.com/open-mmlab/mmdetection/blob/a616886bf1e8de325e6906b8c76b6a4924ef5520/configs/empirical_attention/faster_rcnn_r50_fpn_attention_0010_1x_coco.py#L9

**Comment by BIGWangYuDong** [comment]
You can have a look at: https://github.com/open-mmlab/mmcv/blob/466a6c829c2289651212866bbe24cb3b4541a577/mmcv/cnn/bricks/generalized_attention.py#L34-L42

**Comment by lijoe123** [comment]
Thank you for your reply! 
And, could you tell me how to use the nonlocal-block plugins?Do you have demo?

**Comment by ZwwWayne** [comment]
You can try
```python
model = dict(
    backbone=dict(plugins=[
        dict(
            cfg=dict(
                type='NonLocal2d',
                sub_sample=False,
                conv_cfg=dict(type='Conv2d')),
            stages=(False, False, True, True),
            position='after_conv2')
    ]))
```

**Comment by lijoe123** [comment]
@ZwwWayne By the way, could you please give me a demo about how to use context_block? Thank you!!!!!!
